src/main/java/com/softserve/academy/spaced/repetition/service/AccessToUrlService.java

```diff
@@ -85,6 +85,12 @@ public boolean hasAccessToDeleteCommentForCourse(Long commentId) throws NotAutho
         return hasAccessToUpdateCommentForCourse(commentId) || hasRoleAdmin;
     }
 
+    public boolean hasAccessToUpdateCommentForCourse(Long commentId) throws NotAuthorisedUserException {
```

Comment:
What is the reason to move this method in this pull request? Maybe it worths revert this.

Reply:
yes I agree